test/parallel/test-fs-read-zero-length.js

```diff
@@ -2,17 +2,18 @@
 const common = require('../common');
 const assert = require('assert');
 const path = require('path');
+const Buffer = require('buffer').Buffer;
 const fs = require('fs');
 const filepath = path.join(common.fixturesDir, 'x.txt');
 const fd = fs.openSync(filepath, 'r');
-const expected = '';
+const bufferAsync = Buffer.alloc(0);
+const bufferSync = Buffer.alloc(0);
 
-fs.read(fd, 0, 0, 'utf-8', common.mustCall(function(err, str, bytesRead) {
-  assert.ok(!err);
-  assert.equal(str, expected);
+fs.read(fd, bufferAsync, 0, 0, 0, common.mustCall(function(err, bytesRead) {
   assert.equal(bytesRead, 0);
+  assert.deepStrictEqual(bufferAsync, Buffer.alloc(0));
 }));
 
-const r = fs.readSync(fd, 0, 0, 'utf-8');
-assert.equal(r[0], expected);
-assert.equal(r[1], 0);
+const r = fs.readSync(fd, bufferSync, 0, 0, 0);
+assert.deepStrictEqual(bufferSync, Buffer.alloc(0));
+assert.equal(r, 0);
```

Comment:
I'd prefer to keep test files unrelated to this PR unmodified, since I'm not familiar with their finer details.